nautobot/docs/configuration/authentication/sso.md

```diff
@@ -13,10 +13,15 @@ by default including:
 
 ## Installation
 
+!!! warning
+    This and all remaining steps in this document should all be performed as the `nautobot` user!
+
+    Hint: Use `sudo su - nautobot` 
+
 ### Install Dependencies
 
 ```shell
-pip install social-auth-app-django
+$ pip3 install social-auth-app-django
```

Comment:
Not blocking for this PR, but it'd be great to get these added as optional dependencies so that a user can do `pip install nautobot[sso]` or `pip install nautobot[sso,oidc]` or `pip install nautobot[sso,saml]` or `pip install nautobot[ldap]` and have it automatically pull in the dependencies.

Reply:
Yes, 100% https://github.com/nautobot/nautobot/issues/82